spec/hardware/compatible.fmf

```diff
@@ -0,0 +1,18 @@
+summary:
+    Select guest based on given compatibility
+description:
+    Pick only guests compatible with given operating system
+    distribution. See the context :ref:`/spec/context/dimension`
+    definitions for the list of supported values of the ``distro``
+    key.
+example:
+  - |
+    # Provision a guest supporting the Red Hat Enterprise Linux 8
+    compatible:
+        distro: rhel-8
```

Comment:
I'm not very fond of fields that may be of different types, `str` vs `List[str]`, would "always a list" as a rule be acceptable?

Reply:
I'd say, only list would be also ok.

Reply:
Covered by 1dd30999.